### PR TITLE
refactor(breadcrumb): implement design tokens specs

### PR DESCRIPTION
**Pull request by badejayayesubabu**
## Description

`<breadcrumb>` :  implement design tokens

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF-2200

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: src/scss/components/breadcrumb/index.scss [hunk 1]
```diff
@@ -3,25 +3,28 @@
...
       pointer-events: none;
     }
   }
 
   .delimiter {
-    color: var(--hxBreadcrumb-delimiter-color, $gray-600);
-    font-size: 0.5rem;
-    margin-right: 0.5rem;
+    color: $gray-600;
+    color: $rax-component-breadcrumb-separator-color;
```
Comment (100stacks) [inline src/scss/components/breadcrumb/index.scss:26]: Let's use the same pattern above.

File: src/scss/components/breadcrumb/index.scss [hunk 2]
```diff
@@ -3,25 +3,28 @@
   align-content: center;
   align-items: center;
   color: $blue-500;
+  color: $rax-component-breadcrumb-base-color;
```
Comment (100stacks) [inline src/scss/components/breadcrumb/index.scss:6]: Let's use the IE11 fallback pattern below.

File: src/scss/components/breadcrumb/index.scss [hunk 3]
```diff
@@ -3,25 +3,28 @@
...
     }
   }
 
   .delimiter {
-    color: var(--hxBreadcrumb-delimiter-color, $gray-600);
-    font-size: 0.5rem;
-    margin-right: 0.5rem;
+    color: $gray-600;
+    color: $rax-component-breadcrumb-separator-color;
+    font-size: 0.5rem;//there is no token for font size
```
Comment (100stacks) [inline src/scss/components/breadcrumb/index.scss:27]: Ok, we'll make note of this.

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: src/scss/components/breadcrumb/index.scss [hunk 4]
```diff
@@ -3,25 +3,28 @@
...
   & > a {
-    margin-right: 0.5rem;
+    margin-right: $rax-spacing-200;
     text-transform: capitalize;
 
     &:last-of-type {
-      color: var(--hxBreadcrumb-last-type-color, $gray-800);
-      font-weight: 600;
+      color: $gray-800;
+      color: $rax-component-breadcrumb-current-color;
```
[the remark this answers is not in the document]
Comment (badejayayesubabu) [reply src/scss/components/breadcrumb/index.scss]: Changes have been done, please review.

File: src/scss/components/breadcrumb/index.scss (hunk 1 above)
Comment (badejayayesubabu) [reply src/scss/components/breadcrumb/index.scss]: Changes have been done, please review.

File: src/scss/components/breadcrumb/index.scss (hunk 2 above)
Comment (badejayayesubabu) [reply src/scss/components/breadcrumb/index.scss:5]: Changes have been done, please review.

**Review by 100stacks — APPROVED** [review]
Dev LGTM

**Pull request merged**